vars/beatsWhen.groovy

```diff
@@ -59,63 +60,81 @@ private Boolean whenBranches(Map args = [:]) {
 }
 
 private Boolean whenChangeset(Map args = [:]) {
+  def name = 'Changeset'
   if (args.content?.get('changeset')) {
-    // Gather macro changeset entries
-    def macro = [:]
-    args?.changeset?.each { k,v ->
-      macro[k] = v
-    }
+    def arguments = args
+    arguments.name = name
```

Comment:
I'm not understanding the need for these two lines. Am I just not seeing where `arguments` is used?

Reply:
aha! good catch

Reply:
I've just fixed below, but I'll add some UTs now